src/redux/books/books.js

```diff
@@ -1,13 +1,13 @@
-const ADD_BOOK = 'ADD_BOOK';
-const REMOVE_BOOK = 'REMOVE_BOOK';
-let lastId = 0;
+// Reducers
+
+import * as actions from '../actionTypes';
+import initialData from './initialData';
 
-export const addBook = (book) => ({ type: ADD_BOOK, payload: book });
-export const removeBook = (id) => ({ type: REMOVE_BOOK, payload: { id } });
+let lastId = 0;
 
-function bookReducer(state = [], action) {
+const bookReducer = (state = initialData, action) => {
   switch (action.type) {
-    case 'ADD_BOOK':
+    case actions.ADD_BOOK:
```

Comment:
- [ ] It would be better if you used the `UUID` library to give you a unique id every time you add a new book.